contracts/core/EntryPoint.sol

```diff
@@ -662,9 +662,12 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
                 if (context.length > 0) {
                     actualGasCost = actualGas * gasPrice;
                     if (mode != IPaymaster.PostOpMode.postOpReverted) {
-                        IPaymaster(paymaster).postOp{
+                        try IPaymaster(paymaster).postOp{
                             gas: mUserOp.verificationGasLimit
-                        }(mode, context, actualGasCost);
+                        }(mode, context, actualGasCost)
+                        {} catch (bytes memory reason) {
+                            revert(string(abi.encodePacked("AA96 postOp reverted", reason)));
```

Comment:
this is problematic: you append binary encoding of custom error to a string.
(try to decode the error from the test..)
at a minimum: change string suffix to "...reverted: ", as we do with other reverts

Reply:
Changed to revert with custom error that we can parse off-chain if we add the "postOp reverted" event (see next PR https://github.com/eth-infinitism/account-abstraction/pull/362)